cumulusci/core/config/OrgConfig.py

```diff
@@ -366,3 +374,10 @@ def resolve_04t_dependencies(self, dependencies):
 
             new_dependencies.append(dependency)
         return new_dependencies
+
+
+def _safe_json_from_response(response):
+    try:
+        return response.json()
+    except JSONDecodeError:
+        raise CumulusCIException(f"Cannot decode as JSON:  {response.text}")
```

Comment:
Do you think this is something we would want to use elsewhere? It seems like something we could put in cumulusci.utils...I'm fine waiting to do that until it's actually used in more than one place, as long as we don't forget it's here.